package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "electronic-prescription-service-api",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
not sure where this has come from but I don't think we want it

Reply:
classic